livekit-agents/livekit/agents/worker.py

```diff
@@ -263,7 +265,12 @@ async def _worker_task(self) -> None:
                 if scheme.startswith("http"):
                     scheme = scheme.replace("http", "ws")
 
-                agent_url = f"{scheme}://{parse.netloc}/{parse.path.rstrip('/')}/agent"
+                path_parts = [
```

Comment:
This is unrelated but I was seeing `ws://localhost:7880//agent` in the logs so it's a small fix